src/core/tests/ov_tensor_test.cpp

```diff
@@ -546,7 +818,8 @@ INSTANTIATE_TEST_SUITE_P(copy_tests,
                                                               ov::element::u8,
                                                               ov::element::u16,
                                                               ov::element::u32,
-                                                              ov::element::u64
+                                                              ov::element::u64/*,
+                                                              ov::element::string*/
```

Comment:
why is it commented out?

Reply:
Now I am working on strides fixing for string Tensors. Will do it asap

Reply:
Fixed.